xmac failed  Jep454Tests_testLinkerFfi_DownCall_0
```
        FAILED: test_addDoubleAndFloatFromUnionWithDoubleFloat
        java.lang.AssertionError: expected [235.886] but found [216.666]
```
and Jep454Tests_testLinkerFfi_UpCall_0 crashed
```
21:44:35  ffi_call_unix64+0x52 (0x00000000063F511A [libj9vm29.dylib+0x2a411a])
21:44:35  ffi_call_int+0x3c5 (0x00000000063F4915 [libj9vm29.dylib+0x2a3915])
21:44:35  ffiCallWithSetJmpForUpcall+0xa5 (0x00000000061C58E5 [libj9vm29.dylib+0x748e5])
21:44:35  _ZN32VM_BytecodeInterpreterCompressed3runEP10J9VMThread+0x20b4b (0x00000000062174DB [libj9vm29.dylib+0xc64db])
21:44:35  bytecodeLoopCompressed+0xc8 (0x00000000061F6978 [libj9vm29.dylib+0xa5978])
```
zlinux openjdk failed jdk_lang_0  (known issue)
```
        TEST: java/lang/Thread/virtual/CustomScheduler.java
        Test results: passed: 991; failed: 1; excluded: 97; did not match keywords: 2; did not meet platform requirements: 47

20:48:51  java.lang.InterruptedException
20:48:51  	at java.base/java.lang.Thread.sleepImpl(Native Method)
20:48:51  	at java.base/java.lang.Thread.sleepNanos(Thread.java:515)
20:48:51  	at java.base/java.lang.Thread.sleep(Thread.java:546)
20:48:51  	at CustomScheduler.await(CustomScheduler.java:282)
20:48:51  	at CustomScheduler.testThreadUnparkOOME(CustomScheduler.java:266)
20:48:51  	at java.base/java.lang.reflect.Method.invoke(Method.java:571)
20:48:51  	at java.base/java.util.ArrayList.forEach(ArrayList.java:1604)
20:48:51  	at java.base/java.util.ArrayList.forEach(ArrayList.java:1604)
```
xlinux openjdk failed jdk_lang_0 with a timeout (tracked by https://github.com/eclipse-openj9/openj9/issues/23435)
```
        TEST: java/lang/Thread/virtual/Reflection.java
        Test results: passed: 997; error: 1; excluded: 95; did not match keywords: 2; did not meet platform requirements: 43

19:13:33  ACTION: junit -- Error. "junit" action timed out with a timeout of 960 seconds on agent 28
19:13:33  REASON: User specified action: run junit Reflection 
```

windows openjdk failed jdk_foreign_1
```
        TEST: java/foreign/TestHandshake.java
        Test results: passed: 97; failed: 1; excluded: 11; did not meet platform requirements: 4

06:24:55  Unhandled exception
06:24:55  Type=Segmentation error vmState=0x00040000
06:24:55  Windows_ExceptionCode=c0000005 J9Generic_Signal=00000004 ExceptionAddress=00007FFE1CAA7EC0 ContextFlags=0010005f
06:24:55  Handler1=00007FFE1D203E30 Handler2=00007FFE2406A990 InaccessibleReadAddress=00000217F0C29FB0
06:24:55  RDI=0000000000000000 RSI=0000000000000000 RAX=00000217F0C29FB0 RBX=0000000000000000
06:24:55  RCX=0000000000000001 RDX=00000217F0BAFE90 R8=00000217F0C29FB0 R9=000000000000F424
06:24:55  R10=FFFFFFFFFFF85EE0 R11=0000000000000001 R12=000000000000F424 R13=00000042FE1853F4
06:24:55  R14=00000217F0648600 R15=00000217F0C29FB0
06:24:55  RIP=00007FFE1CAA7EC0 RSP=0000008ECBC7E9B8 RBP=00000042FE175FD0 EFLAGS=0000000000010283
06:24:55  FS=0053 ES=002B DS=002B
06:24:55  XMM0=0000000000000000 (f: 0.000000, d: 0.000000e+00)
06:24:55  XMM1=0000000000000000 (f: 0.000000, d: 0.000000e+00)
06:24:55  XMM2=0000000000000000 (f: 0.000000, d: 0.000000e+00)
06:24:55  XMM3=0000000000000000 (f: 0.000000, d: 0.000000e+00)
06:24:55  XMM4=0000000000000000 (f: 0.000000, d: 0.000000e+00)
06:24:55  XMM5=0000000000000000 (f: 0.000000, d: 0.000000e+00)
06:24:55  XMM6=0000000000000000 (f: 0.000000, d: 0.000000e+00)
06:24:55  XMM7=0000000000000000 (f: 0.000000, d: 0.000000e+00)
06:24:55  XMM8=0000000000000000 (f: 0.000000, d: 0.000000e+00)
06:24:55  XMM9=0000000000000000 (f: 0.000000, d: 0.000000e+00)
06:24:55  XMM10=0000000000000000 (f: 0.000000, d: 0.000000e+00)
06:24:55  XMM11=0000000000000000 (f: 0.000000, d: 0.000000e+00)
06:24:55  XMM12=0000000000000000 (f: 0.000000, d: 0.000000e+00)
06:24:55  XMM13=0000000000000000 (f: 0.000000, d: 0.000000e+00)
06:24:55  XMM14=0000000000000000 (f: 0.000000, d: 0.000000e+00)
06:24:55  XMM15=0000000000000000 (f: 0.000000, d: 0.000000e+00)
06:24:55  Module=C:\Users\jenkins\workspace\Test_openjdk25_j9_sanity.openjdk_x86-64_windows_Personal_testList_1\jdkbinary\j2sdk-image\bin\default\jclse29.dll

06:24:55  ----------- Stack Backtrace -----------
06:24:55  Java_jdk_jfr_internal_JVM_setMiscellaneous+0x4fc0 (0x00007FFE1CAA7EC0 [jclse29+0x57ec0])
06:24:55  Java_sun_misc_Unsafe_copyMemory__Ljava_lang_Object_2JLjava_lang_Object_2JJ+0x208 (0x00007FFE1CA8C938 [jclse29+0x3c938])

06:24:55  TEST RESULT: Failed. Unexpected exit from test [exit code: -1]
```
A few different types of failures for TestHandshake.java are captured here: https://github.com/eclipse-openj9/openj9/issues/13211

--- a/runtime/compiler/optimizer/VectorAPIExpansion.cpp
+++ b/runtime/compiler/optimizer/VectorAPIExpansion.cpp
@@ -370,6 +370,10 @@ void TR_VectorAPIExpansion::visitNodeToBuildVectorAliases(TR::Node *node, bool v
 
             bool aliasTemps = false;
 
+            // Only an aload of an auto can hold a vector value in a temp, so only such a load is aliased
+            // into the temp class to build the web that validateSymRef() later validates.
+            bool isLoadOfAuto = rhs->getOpCodeValue() == TR::aload && rhs->getSymbolReference()->getSymbol()->isAuto();
+
             if (rhs->getOpCode().isFunctionCall()
                 && isVectorAPIMethod(rhs->getSymbolReference()->getSymbol()->castToMethodSymbol())) {
                 // propagate vector info from VectorAPI call to temp
@@ -392,13 +396,13 @@ void TR_VectorAPIExpansion::visitNodeToBuildVectorAliases(TR::Node *node, bool v
                     if (boxingAllowed() && !_nodeTable[rhs->getGlobalIndex()]._canVectorize)
                         dontVectorizeNode(node);
                 } // end of Vector API call as RHS
-            } else if (boxingAllowed() && rhs->getOpCodeValue() != TR::aload) {
+            } else if (boxingAllowed() && !isLoadOfAuto) {
                 dontVectorizeNode(node);
 
                 logprintf(_trace, log, "Making #%d a box of unknown type due to node %p\n", id1, node);
             }
 
-            if (rhs->getOpCodeValue() == TR::aload)
+            if (isLoadOfAuto)
                 aliasTemps = true;
 
             alias(node, rhs, aliasTemps);

